app/pages/project/talk-images.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import getSubjectLocation from '../../lib/get-subject-location.coffee';
+import TalkStatus from './talk-status';
+import Thumbnail from '../../components/thumbnail';
+
+export default class TalkImages extends React.Component {
+
+  render() {
+    return (
+      <div className="project-home-page__container">
+
+        {this.props.images.map((image) => {
+          const subject = getSubjectLocation(image);
+          return (
+            <div key={subject.id} className="project-home-page__talk-image">
+              <Thumbnail alt="" src={subject.src} width="600" height="300" format={subject.format} controls={false} />
```

Comment:
It's complaining about these not being specified as `width="600px" height="300px"` in the console